dts/arm/st/n6/stm32n657X0.dtsi

```diff
@@ -8,7 +8,8 @@
 
 / {
 	axisram1: memory@34000000 {
-		reg = <0x34000000 DT_SIZE_K(624)>;
+		/* 400 kB of FLEXRAM followed by 624 kB of AXISRAM1 */
+		reg = <0x34000000 (DT_SIZE_K(400) + DT_SIZE_K(624))>;
```

Comment:
From ref man, FLEXRAM could be configured to be available at different locations depending on I-TCM and D-TCM configuration.
Since this is defining a static location of the FLEXRAM, shouldn't we check that the configuration is matching this description ?